# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/src/nsFontMetrics.cpp
# Commit: e3f8526e0e2a
# Full Hash: e3f8526e0e2a08a4a8342dccedf05bebb3de1baf
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-09-08 21:39:05
# Regressor Bug: 1715501
# File Overlap Count: 1
# Description:
#   Bug 1715501 - patch 2 - Pass the presContext to platform font lookup methods so they can query it for font visibility. r=emilio
#   
#   This does not in itself change user-visible behavior, but is a foundation for bug
#   1715507 where we will make the behavior per-context instead of global. This is basically
#   just plumbing, passing a pointer to the presContext that's asking for fonts to be
# ==============================================================================

diff -r cfdd1c5d8d3f -r e3f8526e0e2a gfx/src/nsFontMetrics.cpp
--- a/gfx/src/nsFontMetrics.cpp	Wed Sep 08 09:35:16 2021 +0000
+++ b/gfx/src/nsFontMetrics.cpp	Wed Sep 08 09:35:17 2021 +0000
@@ -135,7 +135,7 @@
 
   gfxFloat devToCssSize = gfxFloat(mP2A) / gfxFloat(AppUnitsPerCSSPixel());
   mFontGroup = gfxPlatform::GetPlatform()->CreateFontGroup(
-      aFont.family.families, &style, mLanguage, mExplicitLanguage,
+      mPresContext, aFont.family.families, &style, mLanguage, mExplicitLanguage,
       aParams.textPerf, aParams.userFontSet, devToCssSize);
 }
 